brave/brave-core
# Enable support of Unstoppable Domains and ENS on Android

Resolves https://github.com/brave/brave-browser/issues/15159

![Screenshot_20210415-192044](https://user-images.githubusercontent.com/4730197/114962563-c28b0e80-9e1f-11eb-964b-967db2346cfc.png)

![Screenshot_20210415-191948](https://user-images.githubusercontent.com/4730197/114962557-bf901e00-9e1f-11eb-8a53-df79062ded99.png)

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open settings ->Privacy & Security -> Unstoppable Domains, the settings should be `Ask`.
2. Visit brad.crypto, interstitial page for opting in should be shown, click `Disable`.
3. Go to settings ->Privacy & Security -> Unstoppable Domains, the settings should be `Disabled`.
4. Change it back to `Ask`.
5. Visit brad.crypto again, and click `Proceed` this time in the interstitial page.
6. Page should be loaded and setting should be changed to `DNS over HTTPS`.
Repeat 1-6 but using almonit.eth this time.
7. Go to settings ->Privacy & Security -> Unstoppable Domains, set to Ethereum
8. Visit brave.crypto, the page should be redirected and loaded via IPFS public gateway.
